Automattic/newspack-blocks
# feat(donate): donor name input; initial form hiding

### All Submissions:






### Changes proposed in this Pull Request:



1. UX change in Donation block - form will be hidden at the onset, and then display once the user clicks "Donate"
2. Additional field in the donation form - donor name

### How to test the changes in this Pull Request:

1. Switch to `feat/donations-donor-name` branch of `newspack-plugin`
1. Insert a Donate block, while having "Stripe" set and configured as the donation platform
2. Visit the front-end, observe the form is not initially visible
3. Click "Donate", observe the donation form unravelling
4. Make a donation

### Other information:
